social-media/discord/rules-info-channel.md

```diff
@@ -0,0 +1,45 @@
+# Rules
+
+- Be Kind - This isn't a place for aggression, this is a place to work together to create awesome code.
+- Be Respectful - We aim to have a community of accepting individuals. Being respectful includes not making racist, homophobic, or sexist insults. No matter someone's race, gender, sexuality, political affiliation, or experience we all are here together and should be treated with respect.
+- No Private Messages - Unless confirmed publicly on the server you should never send private unsolicited messages to any members of this server. The only exception would be to report a sensitive issue to the @Admins  or @Moderators. But even then it is recommended to publicly confirm that they are okay with you sending a private message.
+- No Advertising - Please no advertisements of products or services independent of organic conversation.
+- Match your display name to what is on github. This helps for us assigning you to a team on there and allows us to contact you better for updates and concerns on contributions. This is ***NOT*** a strict requirement but it is extremely helpful in operations here.
```

Comment:
```suggestion
- (recommended) Match your display name to what is on github. This helps for us assigning you to a team on there and allows us to contact you better for updates and concerns on contributions. This is ***NOT*** a strict requirement but it is extremely helpful in operations here.
```

I know you do very quickly touch on that this is not a requirement, but I feel like adding a notice right away helps better for anyone who may be skimming this content